README.md

```diff
@@ -304,25 +265,25 @@ fact
 ├── tests
 │   ├── __init__.py
 │   └── test_fact.py
-├── tox.ini
-└── setup.py
+├── noxfile.py
+└── pyproject.toml
 ```
 
 However, this structure
 is [known](https://docs.pytest.org/en/latest/goodpractices.html#tests-outside-application-code) to
-have bad interactions with `pytest` and `tox`, two standard tools maintaining Python projects. The
-fundamental issue is that tox creates an isolated virtual environment for testing. By installing
-the distribution into the virtual environment, `tox` ensures that the tests pass even after the
+have bad interactions with `pytest` and `nox`, two standard tools maintaining Python projects. The
+fundamental issue is that Nox creates an isolated virtual environment for testing. By installing
+the distribution into the virtual environment, `nox` ensures that the tests pass even after the
 distribution has been packaged and installed, thereby catching any errors in packaging and
 installation scripts, which are common. Having the Python packages in the project root subverts
 this isolation for two reasons:
 
 1. Calling `python` in the project root (for example, `python -m pytest tests/`) 
-   [causes Python to add the current working directory](https://docs.pytest.org/en/latest/pythonpath.html#invoking-pytest-versus-python-m-pytest) (
-   the project root) to `sys.path`, which Python uses to find modules. Because the source
-   package `fact` is in the project root, it shadows the `fact` package installed in the tox
-   environment.
-2. Calling `pytest` directly anywhere that it can find the tests will also add the project root
+   [causes Python to add the current working directory](https://docs.pytest.org/en/latest/pythonpath.html#invoking-pytest-versus-python-m-pytest)
+   (the project root) to `sys.path`, which Python uses to find modules. Because the source
+   package `fact` is in the project root, it shadows the `fact` package installed in the Nox
+   session.
+3. Calling `pytest` directly anywhere that it can find the tests will also add the project root
```

Comment:
This does not show up in the rendered file, but if you wanted the text to be numbered correctly:

```suggestion
2. Calling `pytest` directly anywhere that it can find the tests will also add the project root
```